ISPN-13972: Update property realm note in Security realms doc (#10187)
infinispan/infinispan: documentation/src/main/asciidoc/topics/ref_server_realm_property.adoc

[NOTE]
====
You can avoid authentication issues that relate to a property file by using the {brandname} CLI to enter the correct security realm name to the file. 

> 

Reviewer: Please remove the empty lines. In general place each sentence on its own line but don't leave any empty lines between.

Reviewer: This will go to downstream docs once it's merged (the attributes will be substituted from downstream file).
Regarding starting each sentence on its won line - this applies to all our docs. You can check few adoc files and see the format

You can find the correct security realm name of your {brandname} server by opening the `infinispan.xml` file and navigating to the `<security-realm name>` property. 

> If you decide to copy a property file from one {brandname} server to another, you must ensure that the security realm name appropriates to the correct authentication mechanism for the target endpoint. 

Reviewer: I think it can be little shorter, for example
```suggestion
When you copy a property file from one {brandname} Server to another, make sure that the security realm name appropriates to the correct authentication mechanism for the target endpoint. 
```

Author: Updated.
